lib/model_patches.rb

```diff
@@ -6,11 +6,33 @@
 # See http://stackoverflow.com/questions/7072758/plugin-not-reloading-in-development-mode
 #
 Rails.configuration.to_prepare do
-    OutgoingMessage.class_eval do
-        # Add intro paragraph to new request template
-        def default_letter
-            return nil if self.message_type == 'followup'
-            #"If you uncomment this line, this text will appear as default text in every message"    
-        end
-    end        
+  # Remove UK-specific references to FOI
+  InfoRequest.instance_eval do
+
+    THEME_LAW_USED_READABLE_DATA =
+      { :foi => { :short => 'accesso civico',
+                  :full => 'Decreto Trasparenza (D.lgs. 33/2013)',
+                  :with_a => _('A freedom of information request'),
```

Comment:
I will check whether that's the correct phrase in the translation file before rolling it out. Although I don't think it's much used. I'll check that too.